tests/python/test_python_frontend.py

```diff
@@ -1263,7 +1263,16 @@ def fusion_func(fd: FusionDefinition):
 
         with FusionDefinition() as fd:
             fusion_func(fd)
-        assert fd.segment(inputs) == 2
+
+        # create segments
+        fd.segment(inputs)
+
+        # Each segment has a map from its segment index space to the original
+        # fusion's index space. The original fusion creates two segments.
+        # Check that segment maps match expected behavior.
```

Comment:
Can you list the two segments here, so that it is easier to compare the maps below?

Reply:
Additionally, it might be helpful to compare the expected python definitions for the segments similar to `test_repro_script_generation`. Wdyt?

Reply:
I figured you'd ask for a string comparison, but they can be brittle and annoying to maintain. We only care about matching correctness which the next PR will enforce.